test/fixtures/json/search/results_page.json

```diff
@@ -0,0 +1,315 @@
+{
```

Comment:
Should this be in an integration folder? or a page folder to differenciate further from components